sources/Outputs/include/Par.h

```diff
@@ -84,6 +88,7 @@ class Par {
     const algo::ShuntCounterDefinitions& shuntCounters;                           ///< Shunt counters to use
     const algo::DynamicModelDefinitions& dynamicModelsDefinitions;                ///< list of defined dynamic models
     const algo::LinesByIdDefinitions& linesByIdDefinitions;                       ///< lines by id to use
+    algo::StaticVarCompensatorDefinitions svarcsDefinitions;                      ///< the SVarC definitions to use
```

Comment:
why this one not a const Obejc&  like the others ?

Reply:
Corrected